pdf/src/test/groovy/com/craigburke/document/builder/PdfDocumentLoader.groovy

```diff
@@ -68,16 +67,13 @@ class PdfDocumentLoader {
     }
 
     private static void loadChildren(Document document) {
-        def pages = document.element.documentCatalog.allPages
```

Comment:
Ok, I'll take a look at this too. I don't really have a problem with calling a private method in a test if we need to.